cmd/whereabouts_test.go

```diff
@@ -448,6 +448,164 @@ var _ = Describe("Whereabouts operations", func() {
 
 	})
 
+	It("allocates an address using IPRanges notation", func() {
```

Comment:
please squash these unit tests to the commit that changed the production code.

Reply:
Done :+1: